Update admin-submission.md (#6810)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/admin-submission.md

   > [!NOTE]
   > File and URL submissions are not available in the clouds that do not allow for data to leave the environment. The ability to select File or URL will be greyed out.

## Notify users from within the portal

> 1. In the Microsoft 365 Defender portal, go directly to the **Submissions** page: [https://security.microsoft.com/reportsubmission](https://security.microsoft.com/reportsubmission).

Reviewer: ```suggestion
1. In the Microsoft 365 Defender portal, go directly to the [**Submissions**](https://security.microsoft.com/reportsubmission) page.
```

> 2. Click **User reported messages**, and then select the message you want to mark and notify.

Reviewer: ```suggestion
2. Select **User reported messages**, and then select the message you want to mark and notify.
```

> 3. Select the **Mark as and notify** drop-down, and then select **No threats found**, **Phishing**, or **Junk**.

Reviewer: ```suggestion
3. Select the **Mark as and notify** drop-down, and then select **No threats found** > **Phishing** or **Junk**.
```

   > [!div class="mx-imgBorder"]
   > ![Send messages from portal.](../../media/admin-review-send-message-from-portal.png)

> The reported message will be marked as either false positive or false negative, and an email will be automatically sent from within the portal notifying the user who reported the message.

Reviewer: ```suggestion
The reported message will be marked as either false positive or false negative, and an email will be sent automatically from within the portal, notifying the user who reported the message.
```

> The reported message will be marked as either false positive or false negative, and an email will be automatically sent from within the portal notifying the user who reported the message.
> 

Reviewer: ```suggestion
The reported message will be marked as either false positive or false negative and an email will be sent automatically from within the portal to notify the user who reported the message.
```
